recipe/run_test.py

```diff
@@ -15,3 +15,24 @@
 import numpy.random.mtrand
 
 sys.exit(not numpy.test().wasSuccessful())
+
+# for compiled code tests
+if sys.platform == 'darwin':
+    os.environ['LDFLAGS'] = ' '.join((os.getenv('LDFLAGS', ''), " -undefined dynamic_lookup"))
+elif sys.platform.startswith('linux'):
+    os.environ['LDFLAGS'] = ' '.join((os.getenv('LDFLAGS', ''), '-shared'))
+    os.environ['FFLAGS'] = ' '.join((os.getenv('FFLAGS', ''), '-Wl,-shared'))
```

Comment:
You can remove lines 17 - 24

Reply:
I tried with and without them. Same failure.

Reply:
But 17 was copy-n-pasta.

Reply:
https://github.com/conda-forge/numpy-feedstock/pull/99#issuecomment-409690917